src/renderer/Presentational/NodeSettings/WalletSettings.tsx

```diff
@@ -27,13 +27,15 @@ import {
   selectContainer,
   buttonContainer,
   addRow,
+  copyButtonContainer,
 } from './WalletSettings.css';
 import LineLabelSettings from '../../Generics/redesign/LabelSetting/LabelSettings';
 import { Toggle } from '../../Generics/redesign/Toggle/Toggle';
 import DropdownLink from '../../Generics/redesign/Link/DropdownLink';
 import Select from '../../Generics/redesign/Select/Select';
 import Linking from '../../Generics/redesign/Link/Linking';
 import { SettingChangeHandler } from './NodeSettingsWrapper';
+import CopyButton from 'renderer/Generics/redesign/CopyButton/CopyButton';
```

Comment:
I think adding this to your vscode settings will fix your automatic imports to how we have them:
<img width="445" alt="Captura de pantalla 2023-10-22 a la(s) 8 08 27 a m" src="https://github.com/NiceNode/nice-node/assets/3721291/e23534f3-a774-4cff-bbfd-9987471498ae">

or in settings.json `"typescript.preferences.importModuleSpecifier": "relative"`